aiohttp/connector.py

```diff
@@ -375,6 +397,11 @@ def verify_ssl(self):
         return self._verify_ssl
 
     @property
+    def verify_fingerprint(self):
+        """Verify ssl cert fingerprint matches?"""
+        return self._verify_fingerprint
```

Comment:
What's type of the property? Looks like it is `str`, but type of `verify_ssl` is `bool`.
I've found if a bit inconsistent.